tools/admin-console/main.go

```diff
@@ -78,3 +81,71 @@ func main() {
 		log.Fatal(err)
 	}
 }
+
+func funcMap() template.FuncMap {
+	return map[string]interface{}{
+		"deref": deref,
+	}
+}
+
+func deref(i interface{}) (string, error) {
+	switch t := i.(type) {
+	case *string:
+		if t == nil {
+			return "", nil
+		}
+		return *t, nil
+	case *int:
```

Comment:
I wonder if you can just do

`case *int, *int8, *int16, *int32, *uint:`

if all these cases are nearly the same

Reply:
You can't 😦 . `t` is only typed if there's exactly one entry in the `case`. I tried...